hibernate-jpa/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
         exclude group:'org.javassist', module:'javassist'
         exclude group:'org.jboss:jandex', module:'jar'
     }
+    api(libs.managed.hibernate.envers)
```

Comment:
Can you remove this dependency on envers and use strings for the names of the additional types.